test/Directory.Build.props

```
@@ -11,8 +11,10 @@
     <NoWarn>$(NoWarn);SYSLIB0005;SYSLIB0012</NoWarn>
     <!-- Ignore "Naming rule violation: Missing suffix: 'Async'" -->
     <NoWarn>$(NoWarn);IDE1006</NoWarn>
-  </PropertyGroup>
 
+    <!--set this to true to enable the visual runner in the MAUI device tests - make sure it is false for CI-->
+    <EnableMauiDeviceTestVisualRunner>false</EnableMauiDeviceTestVisualRunner>
```

```suggestion
    <EnableMauiDeviceTestVisualRunner Condition="'$(GITHUB_ACTIONS)' != 'true'">true</EnableMauiDeviceTestVisualRunner>
<EnableMauiDeviceTestVisualRunner Condition="'$(GITHUB_ACTIONS)' == 'true'">false</EnableMauiDeviceTestVisualRunner>
```

Do we want to do this? Or are there scenarios where we'd want to disable the visual tests locally?

---

This flag is something I think is best to knowingly turn on/off.  I could force it to be off if CI to safety it, but I've got a note on it

---

I also like to run xharness locally.  The headless runner isn't a perfect runner.